### PR TITLE
docs: update to "deploying" to reduce confusion

### PR DESCRIPTION
**Pull request by Westbrook**
Prep for a new "publishing" section that will relate to getting components up to NPM/webcomponents.org/PIKA as well as this information on getting components/apps onto a live site.

### CODE REVIEW

File: docs/deploying/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ And you're all set up.
 
 ## Example
 
-The [Set-Game Example](https://github.com/open-wc/example-vanilla-set-game/) has the default publishing via storybook on netlify.
+The [Set-Game Example](https://github.com/open-wc/example-vanilla-set-game/) has the deploys its storybook on netlify by default.
```
Comment (thepassle) [inline docs/deploying/README.md:23]: this sentence looks a little weird

Comment (Westbrook) [reply docs/deploying/README.md]: Took another pass.

Comment (thepassle) [reply docs/deploying/README.md]: 👍

**Comment by daKmoR** [comment]
good thinking 👍

**Pull request merged**